packages/forklift-console-plugin/src/modules/Overview/views/overview/tabs/Details/cards/WelcomeCard.tsx

```diff
@@ -1,29 +1,90 @@
 import React, { FC } from 'react';
 import { Trans } from 'react-i18next';
 import automationIcon from 'src/modules/Overview/images/automation.svg';
+import { DetailsItem } from 'src/modules/Providers/utils';
 import { useForkliftTranslation } from 'src/utils/i18n';
 
 import { V1beta1ForkliftController } from '@kubev2v/types';
-import { Card, CardBody, Split, SplitItem, Text, TextVariants } from '@patternfly/react-core';
+import {
+  Card,
+  CardActions,
+  CardBody,
+  CardHeader,
+  Dropdown,
+  DropdownItem,
+  KebabToggle,
+  Split,
+  SplitItem,
+  Text,
+  TextVariants,
+} from '@patternfly/react-core';
 
 type OverviewCardProps = {
   obj?: V1beta1ForkliftController;
   loaded?: boolean;
   loadError?: unknown;
+  onHide?: () => void;
 };
 
-export const OverviewCard: FC<OverviewCardProps> = () => {
+export const OverviewCard: FC<OverviewCardProps> = ({ onHide }) => {
   const { t } = useForkliftTranslation();
+  const actionDropdownItem: any[] = [];
 
+  const [menuIsOpen, setMenuIsOpen] = React.useState(false);
+  const onToggle = () => setMenuIsOpen((open) => !open);
+
+  if (onHide) {
+    actionDropdownItem.push(
+      <DropdownItem
+        key="action"
+        component="button"
+        description={t(
+          "You can always bring this welcome card back into view by clicking 'Show the welcome card' in the page heading.",
+        )}
+        onClick={onHide}
+        data-test="hide"
+        style={{ whiteSpace: 'pre-wrap', width: 280, fontWeight: 'bold' }}
+      >
+        {t('Hide from view')}
+      </DropdownItem>,
```

Comment:
it will be nice to split this chunk into it's own component, 
```ts
if (onHide) { actionDropdownItem.push( <NewComponent /> ) }
```

Reply:
Not sure i understand ??

Reply:
the code is currently formated:
`if () then { big chunk of code }`

the suggestion is to create a new method:
`method = { big chunk of code }`
and then use it in the if:
`if () then { method }`